core/src/main/scala/org/apache/pekko/persistence/r2dbc/ConnectionFactoryProvider.scala

```diff
@@ -40,6 +43,15 @@ object ConnectionFactoryProvider extends ExtensionId[ConnectionFactoryProvider]
 
   // Java API
   def get(system: ActorSystem[_]): ConnectionFactoryProvider = apply(system)
+
+  trait ConnectionFactoryOptionsCustomizer {
```

Comment:
could you review if this trait can be private or `private[r2dbc]`?

Reply:
To use the customizer, users will have to extend the trait in their code. It must be public

Reply:
Apologies - makes sense. Can you add some scaladoc to the trait including `@since 1.1.0`?

Reply:
Np. I've added the scaladoc